pkg/webhooks/resource/handlers.go

```diff
@@ -98,6 +100,8 @@ func NewHandlers(
 		pcBuilder:                    webhookutils.NewPolicyContextBuilder(configuration, jp),
 		admissionReports:             admissionReports,
 		backgroundServiceAccountName: backgroundServiceAccountName,
+		// auditPool:                    pond.New(8, 2000, pond.Strategy(pond.Lazy())),
+		auditPool: pond.New(8, 1000),
```

Comment:
We need to make these configurable.

Reply:
Agreed, tunning them differently impacts stress testing results.

Reply:
Added https://github.com/kyverno/kyverno/pull/10048/files#diff-7b5a7d8ea5301364ac84cdab584c6c04259c697f0e51a2743488fad923ab9424R279-R280.